docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,214 +1,220 @@
-
-Click **NEXT**.
+```code
+$ tctl auth export --type=windows > teleport.cer
+```
 
-The PowerShell script will output a Teleport configuration block. Copy this
-block to a temporary location. Click **Next**.
+Copy this certificate to your Windows system, if you didn't run `tctl` from there.
 
-On the Linux host you installed Teleport to run as the Desktop Access connector,
-edit `/etc/teleport.yaml` and paste the configuration provided by the output of
-the previous step. Review and edit the addition as necessarily for your environment:
+### Install the Teleport service for Windows
 
-```yaml
-version: v3
-teleport:
-  auth_token: 2239...c5b21
-  proxy_server: teleport.example.com:443
+From the Windows system, download the [Teleport service installer](#LinkToCome) {/*Rename as needed */}
+and run it. When prompted, select the Teleport certificate file from the previous step.
+Once complete, reboot the system.
 
-auth_service:
-  enabled: no
-ssh_service:
-  enabled: no
-proxy_service:
-  enabled: no
+<Admonition type="tip" title="Headless Installation">
+The Teleport service installer {/*Rename as needed*/} can be run in a shell
+environment with elevated privileges. You can use this to automate installation,
+uninstallation, and certificate updates. For example:
 
-windows_desktop_service:
-  enabled: yes
-  ldap:
-    addr:     '10.10.1.50:636'
-    domain:   'windows.teleport.example.com'
-    username: 'WIN\svc-teleport'
-    server_name: 'windows-server-hostname'
-    insecure_skip_verify: false
-    ldap_ca_cert: |
-        -----BEGIN CERTIFICATE-----
-        MIIDnzCCAoegAwIBAgIQT/UIn+MT4aZC9ix/QuiV9zANBgkqhkiG9w0BAQsFADBi
-        ...
-        31qA4dO3if7RdikD9hVbiIF9jQ==
-        -----END CERTIFICATE-----
-
-  discovery:
-    base_dn: '*'
-  labels:
-    teleport.internal/resource-id: 42d8859c-60d0-4d7f-9767-bdd66b63fce6
+```code
+$ teleport-windows-auth-setup-v12.0.0-amd64.exe install --cert=teleport.cer -r
+Teleport Authentication Package installed
 ```
 
-Click **Next**.
+Use the `--help` flag to learn more.
 
-Once you've saved `/etc/teleport.yaml`, start Teleport:
+</Admonition>
 
-<Tabs>
-<TabItem label="System Service">
+## Step 2/4. Install the Teleport Desktop Service
 
-This command assumes the `root` user. Prepend `sudo` otherwise.
+On your local system authenticated to your Teleport cluster, generate a
+short-lived join token:
 
 ```code
-$ systemctl start teleport.service
+$ tctl tokens add --type=windowsdesktop
+The invite token: (=presets.tokens.first=)
+This token will expire in 60 minutes.
+
+This token enables Desktop Access.  See https://goteleport.com/docs/desktop-access/
+for detailed information on configuring Teleport Desktop Access with this token.
 ```
 
-Copy the join token to a file on the instance where you will run the Windows Desktop
-Service, and then use the following configuration:
+Copy the token to the Linux host where you will run the Desktop service as `/tmp/token`.
```

Comment:
@ptgott please re-review this section. I realized while responding to other feedback that the instructions lacked a join token for the service node.

Reply:
Added a minor suggestion